app/models/loan.rb

```diff
@@ -2,7 +2,7 @@ class Loan < ApplicationRecord
   belongs_to :item, optional: true
   belongs_to :member
   has_one :adjustment, as: :adjustable
-  has_many :renewals, class_name: "Loan", foreign_key: "initial_loan_id"
+  has_many :renewals, class_name: "Loan", foreign_key: "initial_loan_id", dependent: :destroy
```

Comment:
Without this, some members with renewals can't be deleted using `Member#destroy`.

Reply:
I think it would be nice to have a covering unit test here. I'll push a commit with one! 🛠️

Reply:
16e8792cf8c1cfd4df2edd381e4797cba93dbf67